tests/Functional/BundleInitializationTest.php

```diff
@@ -12,11 +12,10 @@
 namespace Translation\PlatformAdapter\Loco\Tests\Functional;
 
 use Http\HttplugBundle\HttplugBundle;
-use Nyholm\BundleTest\BaseBundleTestCase;
 use Translation\PlatformAdapter\Loco\Bridge\Symfony\TranslationAdapterLocoBundle;
 use Translation\PlatformAdapter\Loco\Loco;
 
-class BundleInitializationTest extends BaseBundleTestCase
+class BundleInitializationTest
```

Comment:
Q: Why was this removed?

Reply:
Because, apparently,  it does not exists anymore.

Reply:
I will try to remove it to check if it's really needed after I fix the other issues.

Reply:
I don't understand why you say this bundle is not needed: this bundle https://github.com/SymfonyTest/symfony-bundle-test seems needed in case we want to test the kernel correctly boots whatever the symfony version with this bundle installed: this seems useful because this bundle supports all symfony version between Symfony 4 and 6.

Also this test is not failing currently because it's not a test anymore, because it's not extending PHPUnit TestCase... But if you extend this PHPUnit TestCase, you will see that it's failing because the method ->createKernel does not exist

Reply:
The Symfony Test Bundle does not work with Symfony 6 https://github.com/SymfonyTest/symfony-bundle-test/tree/1.8.2 -> we'd be forced to use the master branch, instead of a tag; and that's dangerous, for a library.

Reply:
https://github.com/ETSGlobal/loco-adapter/pull/2

Reply:
Actually, since this is a library only used for tests, I guess it's not that dangerous to use the master branch.